src/Teams/Controllers/ManageTeamsController.cs

```diff
@@ -21,5 +28,10 @@ public IActionResult Error()
         {
             return View(new ErrorViewModel { RequestId = Activity.Current?.Id ?? HttpContext.TraceIdentifier });
         }
+
+        public void Remove(int team_id)
```

Comment:
Is this is an action method? If so then we should return IActionResult or Task<IActionResult>